src/components/GameBoard/index.tsx

```diff
@@ -0,0 +1,133 @@
+import './index.scss';
+
+import React, { useContext, ReactNode } from 'react';
+import { PlayerAvatar } from '../Avatar';
+import { GameContext } from '../../Context';
+import isNil from 'lodash.isnil';
+import { Done } from '@material-ui/icons';
+import { PlayingCard } from '../PlayingCard';
+import { Button, Typography } from '@material-ui/core';
+
+const MAX_NUMBER_OF_PLAYERS = 6;
+
+function Fillers(props: { players: number, show: boolean; }) {
+    const { players, show } = props;
+    if (!show) {
+        return null;
+    }
+
+    const shells = new Array(MAX_NUMBER_OF_PLAYERS - players).fill(null).map((_, index) => (
+        <PlayerAvatar
+            key={index + players}
+            player={index + players + 1}
```

Comment:
this is the playerIndex?

Reply:
it's the player index + 1; this is for the styling and positioning of the avatar component